frontend/src/components/Header/Header.js

```diff
@@ -1,13 +1,18 @@
+import { useWindowSize } from "../WindowSizeContext/WindowSizeContext"
 import Announcement from "./Announcement/Announcement"
+import DesktopNavbar from "./Navbar/DesktopNavbar"
 import TitleBanner from "./TitleBanner/TitleBanner"
 
 const Header = () => {
-   const message = "✧ Free shipping on all orders over $15 USD ✧"
+  const message = "✧ Free shipping on all orders over $15 USD ✧"
+  const { width } = useWindowSize()
+  const breakpoint = 640
```

Comment:
Can you move this into `helper.js`?
This number should be relevant whenever we want to do resizes to keep consistent. You can export the breakpoint from the helper function so its consistent across all pages.